pkg/operator/ceph/nfs/config.go

```diff
@@ -56,7 +56,7 @@ NFS_CORE_PARAM {
 	Protocols = 4;
 }
 
-CACHEINODE {
+MDCACHE {
```

Comment:
When was that introduced? Which release?

Reply:
As of nfs-ganesha v2.8